roles/jd-client/src/lib/error.rs

```diff
@@ -209,12 +210,6 @@ impl<'a>
     }
 }
 
-impl<'a> From<Vec<u8>> for Error<'a> {
```

Comment:
I assume this is being removed because it's no longer explicitly used in the deserialization process and started triggering build warnings?

is that correct?

Reply:
yes